app/styles/main.scss

```diff
@@ -1,2 +1,11 @@
 @import 'variables';
-@import 'general';
+@import 'mixins';
+//@import 'breakpoints';
+@import 'base';
+@import 'layout';
+@import 'modules';
+@import 'state';
+//this file still needs to go
+// will be replaced by _breakpoints.scss
+@import 'media_queries'; 
+
```

Comment:
The aforementioned **sass_globbing** trick wouldn't work here since I think these are in a very specific order.